dev-python/txtorcon/Manifest

```diff
@@ -0,0 +1,4 @@
+DIST txtorcon-20.0.0.tar.gz 306139 BLAKE2B 54474ef6facf5c54ccc9feed0e8c5b58b3f5356b73290f7d0d488910637e541c2e4d7b8c90470aa561f6bcc202532e153761e9409375d102d9c68473cf34a456 SHA512 4856c86b3eac432b8e2dddec4d4ef3172452d084a5685f90335d366e918e2e80a223a7a31ff78ddb6dff1ddbbfc1e26f6d0ae9fb99bd47d6894cc6dd29365da8
+DIST txtorcon-21.1.0.tar.gz 310839 BLAKE2B bc9c44485e157bce8acdcfa499d45280f37712ad99d328fbc07cefccf012a53e8b6d45537c4b3335073f9fcfbc37ab992b9d81224e2050622e6345283dffa48d SHA512 1e8b1435d7bdf8cf1a9e64b9de6fe2c266ca9812d04405184a117b0b2549677f9881ac373a24d867d3d72e5c7771426de0a7a95e5b9361c9489d811c2a46c343
+EBUILD txtorcon-20.0.0.ebuild 1797 BLAKE2B 6c11484b0800771c9654064b0a1e60acc374fc6c913d44d0154137b854057123b75c3853b454fe8ff104e10cc955593dcf761730c1b6c543f3c83b3d0620a21e SHA512 5c381297134cefe9b67e71f3f9ba8b94bbf96fe33082e681dc44b5605a9cf15f30543592b2a82256d7d18ac4d5f736d4e97d914da20fdc1cfaa7d18a37c383d5
+EBUILD txtorcon-21.1.0.ebuild 1796 BLAKE2B badcf2e059b121f9e18502623f6c047db2039527824a025363957c6688212d92ff7aef9063a1af82168b93f21e962c734664218aa48bb14c76287b88a4f6a8b1 SHA512 dfcaf70ac9ffb3a274e71778f5e1db2c57bcfc7ed37c295c600008a583e3a49165eeaaeeefc430e4e9664eaec49d8cf5ff0325e62128ca6265c752631f52ff48
```

Comment:
need to fix manifests / use thin-manifests in your overlay (it's not necessary with git really)